src/app/services/config/editor.config.json

```diff
@@ -284,5 +284,6 @@
     "FlagDraft",
     "Live"
   ],
-  "reviewer": ["Review"]
+  "reviewer": ["Review"],
+  "publicStorageAccount": "https://sunbirddev.blob.core.windows.net/"
```

Comment:
Why is it hardcoded the value.

This should be environment variable, so that it will be changes based on dev or staging etc..

Reply:
@vinukumar-vs Resolved.